Src/AutoFixture.NUnit3.UnitTest/Scenario.cs

```diff
@@ -295,5 +295,52 @@ public void FreezeSecondParameterOnlyFreezesSubsequentParameters(Guid g1, [Froze
         {
             Assert.AreNotEqual(p1, p2.Field);
         }
+
+        [Theory]
+        [InlineAutoData(1, 2, 3)]
+        public void InlineAutoDataTakesParameterValues(int p1, int p2, int p3)
+        {
+            Assert.That(1, Is.EqualTo(p1));
+            Assert.That(2, Is.EqualTo(p2));
+            Assert.That(3, Is.EqualTo(p3));
+        }
+
+        [Theory]
+        [InlineAutoData]
+        public void InlineAutoDataProvidesParameterValuesWhenNoneGiven(int p1, int p2, int p3)
+        {
```

Comment:
I had doubts too when I basically copied the very 1st test from the Scenarios

`
        [Test, AutoData]
        public void AutoTestCaseProvidesCorrectInteger(int primitiveValue)
        {
            Assert.AreNotEqual(0, primitiveValue);
        }

`

Any clarifications?